resources/language/resource.language.pl_pl/strings.po

```diff
@@ -220,13 +220,13 @@ msgstr "{} Ustaw do odtwarzania z biblioteki"
 # Unused 30053, 30054
 msgctxt "#30055"
 msgid "{} Set at start-up"
-msgstr ""
+msgstr "Ustaw przy uruchomieniu"
 
 #. The {} will be replaced with the graphic circle to mark when the functionality is active
 # Unused 30056
 msgctxt "#30057"
 msgid "{} Remember PIN"
-msgstr ""
+msgstr "Zapamiętaj PIN"
```

Comment:
placeholder is missing

Reply:
I corrected the translation, added the missing translations, but it was not verified. I don't know where the problem is?

Reply:
its ok now,
CI pylint its not a translation problem